backbone/src/main/java/org/researchstack/backbone/step/Step.java

```diff
@@ -21,7 +22,7 @@
  * To implement a new type of step, subclass Step and add your additional properties. Separately,
  * subclass StepLayout and implement your user interface.
  */
-public class Step implements Serializable {
+public class Step implements Serializable , Cloneable{
```

Comment:
There should be a space after Cloneable, spots Bug will complain if you don't change it, I think.